services/spark-worker.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "spark-worker",
+  "description": "Spark Worker",
+  "dependencies": {
+    "conflicts": [],
+    "install": {
+      "requires": [ "base" ],
+      "uses": []
+    },
+    "provides": [],
+    "runtime": {
+      "requires": [],
+      "uses": []
+    }
+  },
+  "provisioner": {
+    "actions": {
+      "install": {
+        "type":"chef-solo",
+        "fields": {
+          "run_list": "recipe[hadoop::spark_worker]"
+        }
+      },
+      "configure": {
+        "type": "chef-solo",
+        "fields": {
+          "run_list": "recipe[hadoop_wrapper::default],recipe[hadoop::spark]"
```

Comment:
same as above, this should be ``hadoop::spark_worker`` instead of just ``hadoop::spark``